recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
 {% set name = "scikit-surprise" %}
 {% set version = "1.1.3" %}
-{% set sha256 = "56bfec410b21278807b7538acb7405db99b44937259dd329762cee7e667d4bcc" %}
+{% set numpy = "1.19.5" %}  # [py<=39 and not (osx and arm64)]
+{% set numpy = "1.21" %}  # [py<=39 and (osx and arm64)]
+{% set numpy = "1.21" %}  # [py>39 and py<311]
+{% set numpy = "1.23" %}  # [py>=311]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 56bfec410b21278807b7538acb7405db99b44937259dd329762cee7e667d4bcc
 
 build:
-  number: 1
-  script: python setup.py install --single-version-externally-managed --record record.txt
-
+  number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - surprise = surprise.__main__:main
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
-    - setuptools
```

Comment:
is in't setuptools mandatory in host section, linter would not complain ?

Reply:
The linter complains about the `run` section. In the host section it's below https://github.com/AnacondaRecipes/scikit-surprise-feedstock/pull/2/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL27

Reply:
remove from run and will be ok for me

Reply:
setuptools is required https://github.com/AnacondaRecipes/scikit-surprise-feedstock/pull/2/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR37